partner_fax/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# © 2018 Apruzzese Francesco <f.apruzzese@apuliasoftware.it>
```

Comment:
Please avoid Copyright from `__init__` file according to https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py

Reply:
Dropped it, thanks.